packages/blocks/src/components/question-wrapper/index.js

```diff
@@ -24,6 +25,17 @@ const StyledQuestionWrapper = styled.div`
 	&.has-box-shadow {
 		box-shadow: 2px 2px 8px rgba( 0, 0, 0, 0.23 );
 	}
+
+	&.is-error {
```

Comment:
I believe this has the potential to create some issues:

- If the question's `border-width` is set to `0`, the error border will not show up. 
- While the above is somewhat acceptable, if the question background is then set to transparent, the error messages can look quite confusing.

I think we need to find a way to force the border when an error occurs.

Reply:
Will handle those style changes on c/juGexWOV-tr